en/docs/administer/review-workflow-approval-requests.md

```diff
@@ -0,0 +1,29 @@
+# Review Workflow Approval Requests
+
+In Choreo, administrators can [configure approvals for workflows](./configure-approvals-for-choreo-workflows.md) and designate authorized assignees to review approval requests.
+
+If you are assigned as an authorized reviewer for a particular workflow approval request, you will receive an email notification whenever a [request is submitted for approval](../develop-components/request-for-workflow-approval.md). The email will include a summary of the request and a link to the **Approvals** page in the Choreo Console, where you can review the details and either approve or reject the request.
+
+!!! note 
+     - Other members in the organization may also receive the same notification and might have already reviewed the request. In such cases, you will not be able to review it again.
```

Comment:
`Other members in the organization`. In fact not any member.  It should be one configured to approve workflows

Reply:
Done